metricbeat/module/elasticsearch/node/node.go

```diff
@@ -64,23 +63,16 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 // Fetch methods implements the data gathering and data conversion to the right format
 // It returns the event which is then forward to the output. In case of an error, a
 // descriptive error must be returned.
-func (m *MetricSet) Fetch(r mb.ReporterV2) {
+func (m *MetricSet) Fetch(r mb.ReporterV2) error {
 	content, err := m.HTTP.FetchContent()
 	if err != nil {
-		elastic.ReportAndLogError(err, r, m.Log)
-		return
+		return err
```

Comment:
Maybe give the error more info here like errors.Wrap(err, "failed to fetch content")?

Reply:
@ycombinator Perhaps you can take it up in an other PR?